packages/core/src/codewhispererChat/controllers/chat/telemetryHelper.ts

```diff
@@ -188,6 +192,38 @@ export class CWCTelemetryHelper {
                     cwsprChatHasProjectContext: this.responseWithProjectContext.get(message.messageId),
                 }
                 break
+            case 'acceptDiff':
+                message = message as ViewDiff
+                event = {
+                    result: 'Succeeded',
+                    cwsprChatConversationId: conversationId ?? '',
+                    cwsprChatMessageId: message.messageId,
+                    cwsprChatInteractionType: 'acceptDiff',
+                    credentialStartUrl: AuthUtil.instance.startUrl,
+                    cwsprChatAcceptedCharactersLength: message.code.length,
+                    cwsprChatHasReference:
+                        message.referenceTrackerInformation && message.referenceTrackerInformation.length > 0,
+                    cwsprChatCodeBlockIndex: message.codeBlockIndex,
+                    cwsprChatTotalCodeBlocks: message.totalCodeBlocks,
+                    cwsprChatHasProjectContext: this.responseWithProjectContext.get(message.messageId),
+                }
+                break
+            case 'view_diff':
```

Comment:
nit: seems strange that this uses `snake_case`, `acceptDiff` is camelCase and the others are `hyphen-words`

Reply:
Indeed thats very true. Even during development its weird. I tried keep it same as existing syntaxes. This is a good avenue to normalize strings. Do you have a specific recommendation?